plugin/src/main/java/org/opensearch/sql/plugin/SQLPlugin.java

```diff
@@ -342,9 +342,6 @@ public Collection<SystemIndexDescriptor> getSystemIndexDescriptors(Settings sett
     systemIndexDescriptors.add(
         new SystemIndexDescriptor(
             SPARK_REQUEST_BUFFER_INDEX_NAME + "*", "SQL Spark Request Buffer index pattern"));
-    systemIndexDescriptors.add(
-        new SystemIndexDescriptor(
-            OpenSearchAsyncQueryScheduler.SCHEDULER_INDEX_NAME, "SQL Scheduler job index"));
```

Comment:
Should we remove the constant (`SCHEDULER_INDEX_NAME`) as well?

Reply:
no, it's still in use https://github.com/opensearch-project/sql/pull/3092/files#diff-b9ca34b799db4aa7b3c7dbccba65de6f539a5d3d6accf8a1c75e4c6893b9a9b8R264